utils/test-deployment/run_tests.py

```diff
@@ -51,9 +51,12 @@
 if not SERVER_VERSION:
     SERVER_VERSION = os.getenv('SERVER_VERSION')
 if SERVER_VERSION:
-    assert server_info.version == SERVER_VERSION, f"The deployed version {server_info.version} doesn't match the expected {SERVER_VERSION}"
-    print(f"Server version {SERVER_VERSION} is deployed and available")
+    if not SERVER_VERSION == 'latest':
```

Comment:
Test blew up when I was using image `latest`, so this is an attempt at Python to fix that.